static/styles/settings.css

```diff
@@ -1092,17 +1092,34 @@ input[type="checkbox"] {
         border-radius: 5px;
 
         h4 {
+            .name {
+                font-weight: 1000;
+            }
+
             font-weight: normal;
-            margin: 0;
-            display: flex;
-            align-items: center;
-            justify-content: left;
+            display: grid;
+            grid-template-columns: 5fr 1fr;
+            gap: 5px;
+
+            .buttons-container {
+                margin-left: auto;
+                display: flex;
+                justify-content: flex-end;
+                width: 140px;
+
+                button {
+                    height: 35px;
+                    width: 35px;
+                    margin: 0 2px;
+                }
+
+                .sea-green {
```

Comment:
It would be better to use `save-status` class for more clarity.

Reply:
The ```Delete``` button also has the same class. Using ```save-status``` would change the width of the ```Delete``` button.

Reply:
We should remove the `save-status` class from delete button as it is not correct anyway.

Reply:
Yeah sure.